### PR TITLE
[Main Course] - pdf crawler

### PR DESCRIPTION
**Pull request by jhqian0909**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Post for pdf crawler

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by BulletTech2021 — COMMENTED** [review]
个人推荐强调解决URL不变的网站，处理爬虫问题的解决思路。 在原本的叙述基础上可以进行一些总结，归纳

File: docs/Main_Course/Data_Analytics/2021-07-01-pdf-Crawler.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+```html
+http://icid.iachina.cn/ICID/front/leafColComType.do?columnid=2016072012158397
+```
+
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/2021-7-1/1625140634123-1.jpg" width="500" />
+  <figcaption>Real url</figcaption>
+</figure>
+        
+我们进入此url。完蛋，这只是一个和上一页面神似但是长的更丑的网址。
```
Comment (BulletTech2021) [inline docs/Main_Course/Data_Analytics/2021-07-01-pdf-Crawler.md:28]: 长的 - 》长得。

File: docs/Main_Course/Data_Analytics/2021-07-01-pdf-Crawler.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+---
+
+# 爬取并下载url不变的pdf文件
+
+!!! info 
+    作者：Void，发布于2021年6月29日，阅读时间：约10分钟，微信公众号文章链接：[:fontawesome-solid-link:](https://mp.weixin.qq.com/s/3JlO0eO95SBqhHH4EHPypw)
+
+## 1 引言
+
+为了更好的卖保险(导师要求)，需要下载[保险业协会官网](http://icid.iachina.cn/?columnid_url=201510010005#)-信息披露中保险公司披露的pdf文件。保险公司很多，每家又有不少年度披露的pdf。同时，神奇的是，无论怎么点击页面，网页的url都没有发生变化。为了拒绝当人肉爬虫，我们再次尝试使用Python帮助我们高效、自动的下载这些pdf文件。
```
Comment (BulletTech2021) [inline docs/Main_Course/Data_Analytics/2021-07-01-pdf-Crawler.md:12]: 自动的 -》 自动地

**Review by jz263 — APPROVED** [review]
LGTM

File: docs/Main_Course/Data_Analytics/2021-07-01-pdf-Crawler.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+!!! info 
+    作者：Void，发布于2021年6月29日，阅读时间：约10分钟，微信公众号文章链接：[:fontawesome-solid-link:](https://mp.weixin.qq.com/s/3JlO0eO95SBqhHH4EHPypw)
+
+## 1 引言
+
+为了更好的卖保险(导师要求)，需要下载[保险业协会官网](http://icid.iachina.cn/?columnid_url=201510010005#)-信息披露中保险公司披露的pdf文件。保险公司很多，每家又有不少年度披露的pdf。同时，神奇的是，无论怎么点击页面，网页的url都没有发生变化。为了拒绝当人肉爬虫，我们再次尝试使用Python帮助我们高效、自动的下载这些pdf文件。  
+
+## 2 具体步骤
+
+我们打开保险业协会网站，点击不同的科目，如保险公司年度信息披露，我们发现页面的url并没有发生变化。这时，请不要怀疑自己的眼睛或是砸烂电脑，我们应该合理的怀疑页面采取了某些异步请求(Ajax)的方式。
```
Comment (BulletTech2021) [inline docs/Main_Course/Data_Analytics/2021-07-01-pdf-Crawler.md:16]: 合理的 -》 合理地

File: docs/Main_Course/Data_Analytics/2021-07-01-pdf-Crawler.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+
+中，使用select an element去审查每家保险公司的名字，发现其简称都存储在控件a的id中。因此，我们可以通过遍历的方式得到所有保险公司的简称，并带入到comCode中。
+以国寿资产(GSZC)为例，我们进入新的网址：
+
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/2021-7-1/1625140619330-3.jpg" width="500" />
+  <figcaption>Example</figcaption>
+</figure>
+
+其中每一个pdf就是我们最终想得到的结果了，我们点击一个公告，同样查看它的XHR
```
Comment (BulletTech2021) [inline docs/Main_Course/Data_Analytics/2021-07-01-pdf-Crawler.md:56]: 末尾少了一个标点

**Review by BulletTech2021 — APPROVED** [review]
有个别错别字，新的修改让文章清楚了许多！

**Pull request merged**